lib/CMakeLists.txt

```diff
@@ -37,7 +39,8 @@ set_property(TARGET jetpeer APPEND PROPERTY PUBLIC_HEADER
 
 # Synchronous peer is a shell around the asynchronous peer
 # Programs using the Synchronous peer, have also to link the asynch peer
-target_link_libraries(jetpeer INTERFACE jetpeerasync)
+# It also requires multithreading
+target_link_libraries(jetpeer INTERFACE jetpeerasync ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
As of cmake 3.1, the better way to do this is:

`target_link_libraries(jetpeer INTERFACE jetpeerasync Threads::Threads)`

https://cmake.org/cmake/help/latest/module/FindThreads.html

It's no longer necessary to refer directly to the "low level" output variables of `FindThreads()`. The interface target will bring in not only the correct libraries but any additional compiler/linker flags that might be needed on some platforms.